app/Libraries/Opengraph/CommentOpengraph.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+namespace App\Libraries\Opengraph;
+
+use App\Models\Comment;
+
+class CommentOpengraph
+{
+    public function __construct(private Comment $comment)
+    {
+    }
+
+    public function get(): array
+    {
+        $user = $this->comment->user;
+
+        return priv_check('CommentShow', $this->comment)->can()
+            ? [
+                'description' => html_excerpt($this->comment->message_html, 100),
```

Comment:
`blade_safe`? return of `html_excerpt` is already escaped

although I noticed discord double unescape description by default 🤔 and twitter is even worse